eox_core/api/v1/serializers.py

```diff
@@ -71,6 +76,48 @@ def validate(self, attrs):
         return attrs
 
 
+class WrittableEdxappUserSerializer(EdxappUserSerializer):
+    """
+    Handles the serialization of the user data required to update an edxapp user.
+    """
+    password = serializers.CharField(
+        style={'input_type': 'password'},
+    )
+    is_active = serializers.BooleanField()
+
+    def validate(self, attrs):
+        """
+        When an update is being made, then it checks that the user:
+            - Is not staff or superuser.
+            - Has just one signup source.
+            - The field being updated is a safe field.
```

Comment:
I dont see this validation that the field is safe. Is it just that the key exists in this serializer?
Could I by mistake update the username using this?

Reply:
That was what I had before. I got confused when we talked about adding the new serializer. 
I'm going to add the following:
SAFE_FIELDS_TO_UPDATE = ["is_active", "password", "fullname"]
should I add another field?

Reply:
ok nice, I'll add the setting and put those fields as defaults